src/services/ros/realms.ts

```diff
@@ -82,6 +87,31 @@ export const open = async (params: {
   return realm;
 };
 
+const getPathOnDisk = (
+  user: Realm.Sync.User,
+  realmPath: string,
+): string | undefined => {
+  if (process.platform === 'win32') {
+    const result = path.join(
+      user.identity,
+      crypto
+        .createHash('md5')
+        .update(realmPath)
+        .digest('hex'),
+    );
+
+    fs.ensureDirSync(path.join(process.cwd(), user.identity));
+
+    // tslint:disable-next-line:no-console
+    console.log(
+      `Rewrote: '${user.identity} - ${getUrl(user, realmPath)}' to '${result}'`,
+    );
```

Comment:
Do we really need this? I suggest removing it.

Reply:
I think it's useful to know the name of the file that corresponds to a url. An alternative approach I was considering was to add a menu item in the browser to locate the file on disk, but that seemed too much for something so simple. I think it's mostly useful for debugging purposes (e.g. to see how large the client realm is after initial sync), so opening the debug tools and checking the console output seemed like the best tradeoff between preserving the functionality and not overengineering it.

Reply:
In its current form Realm Studio is writing pretty much nothing to the console (except for Electron and Realm which writes directly to stdout), hence the no-console lint rule.

In my experience it's easier to instrument the app with console.logs when you're debugging a particular issue (or attach a debugger?) then to graciously put them anywhere they might come in handy. You'll always miss that thing you needed anyway.

Perhaps we should add an easy way to get the size on disk of the client-side Realm instead?

Reply:
I feel strongly there should be a way for us to reverse map a hashed path to a file on disk. Console.logging the path was the cheapest way to achieve that, considering Realm devs would be able to open the developer tools and inspect the console output. If you prefer to achieve that goal in a different way, feel free to contribute to the PR and expose menu options.